doc/neps/nep-0048-spending-project-funds.rst

```diff
@@ -0,0 +1,421 @@
+.. _NEP48:
+
+=====================================
+NEP 48 — Spending NumPy Project Funds
+=====================================
+
+:Author: Ralf Gommers <ralf.gommers@gmail.com>
+:Author: Inessa Pawson <inessa@albuscode.org>
+:Author: Stefan van der Walt <stefanv@berkeley.edu>
+:Status: Draft
+:Type: Informational
+:Created: 2021-02-07
+:Resolution:
+
+
+Abstract
+--------
+
+The NumPy project has historically never received significant **unrestricted**
```

Comment:
But did it receive restricted funding, was "significant" or "unrestricted" the problem before ? Do we need to invoke history before its definition here?

Reply:
> But did it receive restricted funding, 

Yes, NumPy has received restricted funding. For details, refer to the section “History and current status” below.

In case you are not familiar with the internals of non-profit and academic funding, **restricted funds** are monies set aside for a particular purpose (e.g., project, educational program, etc.). They are **permanently restricted** to that purpose and cannot be used for any other expenses.  By contrast, unrestricted funds may be used for any legal purpose appropriate to the organization.

> was "significant" or "unrestricted" the problem before ? 

The lack of **unrestricted funds** was, and still remains, the biggest challenge.

> Do we need to invoke history before its definition here?

Initially, the History and Current Status section was a part of the Motivation and Scope. Its large wordcount made us concerned that it would be taking away readers’ attention from the objectives of this document. Hence, we moved it to the end. 
Would it help to insert a sentence at the beginning of this section linking to the section below? E.g., _For the overview of the NumPy funding history and current status please refer to [NumPy funding – history and current status](url)._